libfreerdp/crypto/crypto.c

```diff
@@ -843,7 +843,7 @@ rdpCertificateData* crypto_get_certificate_data(X509* xcert, const char* hostnam
 
 	issuer = crypto_cert_issuer(xcert);
 	subject = crypto_cert_subject(xcert);
-	certdata = certificate_data_new(hostname, port, issuer, subject, fp);
+	certdata = certificate_data_new(hostname, port, subject, issuer, fp);
```

Comment:
While this change might be correct it will break comparison with existing known_hosts files.

Reply:
Done, reverted. But this row so confusing)

Reply:
yes, old typo but does not matter internally (but would screw over all old entries)